### ISSUE TITLE
Should be Output spammed by messages/errors?

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#24185]
**Godot version:**
3.1 89f8b84 and 3.0.6

**OS/device including version:**
Windows 10

**Issue description:**
It is annoying when Output are spammed by even hundreds of same messages.

Simple changing self modulate of CanvasItem node, show in input a lot of messages which are unnecessary in such a quantity, Should be enough if message appear only once when we change a value(e.g changing modulate by dragging dragging from 170 -> 180 should show 1 Output message instead like now 10).

Also when I create bad polygon, each move cause show in Output more than one error message. I see in 2D viewport that this polygon isn't shown correctly so it isn't necessary to show so much errors.

Probably there is many more cases when one message should be enough.

**Steps to reproduce**

![jfile](https://user-images.githubusercontent.com/41945903/70704326-1177df80-1cd2-11ea-9c30-7d37613c16d8.gif)

![fff](https://user-images.githubusercontent.com/41945903/49569281-cc3d0700-f933-11e8-805d-480b1383808f.png)

**Comment by pwab** [comment]
This is still relevant but should be moved to the [GIP](https://github.com/godotengine/godot-proposals).
Related to https://github.com/godotengine/godot-proposals/issues/971

**Comment by akien-mga** [comment]
Closing as superseded by godotengine/godot-proposals#971.
